src/App.test.js

```diff
@@ -2,8 +2,11 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import App from './App';
 
+// 'test' and 'expect' are not defined
+// eslint-disable-next-line
```

Comment:
probably add "jest" to eslint globals instead of disabling. we'll want to have this far far the the future

Reply:
Added to globals